test/functional-portable/corerp/noncloud/resources/testdata/containers/corerp-resources-friendly-container-version-1.bicep

```
@@ -41,7 +41,7 @@ resource redisContainer 'Applications.Core/containers@2023-10-01-preview' = {
   properties: {
     application: app.id
     container: {
-      image: 'redis:6.2'
+      image: 'ghcr.io/radius-project/dev/redis:6.2'
```

We're purging the images which has `dev/` prefix regularly by [this workflow](https://github.com/radius-project/radius/blob/5c50b81184cd117df90c6fdf5e25f3e904507030/.github/workflows/purge-test-resources.yaml#L38-L45). So I would not recommend to use `dev/`

I would choose the different path name like below.

 ```suggestion
      image: 'ghcr.io/radius-project/mirror/redis:6.2'
```